components/breadcrumb/src/index.js

```diff
@@ -106,13 +90,17 @@ const Breadcrumb = ({ children, ...props }) => (
   </BreadcrumbContainer>
 );
 
-Breadcrumb.propTypes = {
+BreadcrumbComponent.propTypes = {
   /**
    * Breadcrumb contents
    */
   children: PropTypes.node.isRequired,
 };
 
+const Breadcrumb = deprecate(BreadcrumbComponent, 'please use the Breadcumbs component instead');
+
 Breadcrumb.Link = Link;
 
+export { BreadcrumbComponent };
+
 export default Breadcrumb;
```

Comment:
if this is just a rename, can't we import the BreadCrumbs and wrap it with deprecate?

Reply:
Not quite.

I followed the path of least resistance on this - which has indeed left us with more code than we really need.

The main problem with going for a simpler option of importing the new `Breadcrumbs`, deprecating it, and exporting that would be that it wouldn't be documented.